recipes/earthpy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "earthpy" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f604ffe3666d59f4104bff2ca887312c4c7b827a87f8716561ae86b5e9b76a05
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - geopandas
+    - matplotlib >=2
+    - numpy
+    - pandas
+    - rasterio
+    - Rtree>=0.8
+    - scikit-image
+
+
+test:
+  imports:
+    - earthpy
+    - earthpy.clip
+    - earthpy.io
+    - earthpy.mask
+    - earthpy.plot
+    - earthpy.spatial
+
+about:
+  home: http://github.com/earthlab/earthpy
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The build is failing because the licence file isn't included in the tarball on PyPI (which is technically in breach of the licence).

You can fix it by downloading [this file](https://github.com/earthlab/earthpy/blob/master/LICENSE) and adding it to the recipe directory.

Reply:
Thank you @chrisburr !! i think we are also including a manifest. file to ensure the license is bundled properly. @mbjoseph  is working on it! i just accepted the PR